eng/versioning/version_client.txt

```diff
@@ -101,7 +101,7 @@ com.azure:azure-core-test;1.26.2;1.27.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.48;1.0.0-beta.49
 com.azure:azure-core-tracing-opentelemetry-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-version-tests;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos;4.63.2;4.63.3
+com.azure:azure-cosmos;4.63.2;4.63.3-hotfix
```

Comment:
Shouldn't the version be 4.63.4? I saw in the changelog the version still increased with hotfixes.

Reply:
You're right - fixed it in next iteration.